.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 # DB
 *.sqlite3
 db
+
+# MacOS-specific binary file
+.DS_Store
```

Comment:
IIRC, `.DS_Store` tells macOS's finder how to render the files in the containing directory. You'll want to change this entry to `*.DS_Store` to exclude `.DS_Store` files from all directories, including subdirectories. (This entry just excludes the top-level directory's `.DS_Store`.)